netbox/circuits/tables.py

```diff
@@ -140,6 +140,9 @@ class CircuitTable(BaseTable):
         template_code=CIRCUITTERMINATION_LINK,
         verbose_name='Side Z'
     )
+    install_date = tables.DateColumn(
+        default=''
```

Comment:
Unfortunately this approach removes the default placeholder (&mdash;) in the table when rendered in the UI, which we want to keep.

It looks like DateColumn inherits from TemplateColumn: The ideal fix for this might actually be to fix it upstream in django-tables2 by adding a `value()` method to DateColumn to better handle null values.

Reply:
Yes, I agree in that it might be better to fix it upstream. This behavior will occur again in the case of exporting other null date fields, like date_added in the Aggregate model. 

I can go and open an issue there.

Reply:
The django-tables2 library doesn't seem to very active at the moment.

What do you think of implementing a custom DateColumn and improving the handling of null values within the project? 

It could be added into the utilities/tables.py module.

Reply:
I'd rather avoid that, but it was be a reasonable short-term fix. Do you want me to keep #8391 assigned to you?

Reply:
I don't like it too much either, but sure it can be a short-term fix. Yes please keep it assigned to me, I can work on it this weekend.

Reply:
Thanks!

Reply:
This is the best way I found to handle null values for all fields of type Date and DateTime, feel free to give comments on the approach.

Reply:
Looks great to me! Thanks for your work.